R/render_site.R

```diff
@@ -72,7 +72,7 @@ clean_site <- function(input = ".", preview = FALSE, quiet = FALSE,
   else {
     if (!quiet) {
       cat("Removing files: \n")
-      cat(paste(paste0(" ", files), collapse = "\n"))
+      cat(paste0(" ", files), sep = "\n"))
```

Comment:
@yihui sure, this makes sense. Looks like you have an extra `)` though!! Should be:

```
cat(paste0(" ", files), sep = "\n")
```

Reply:
You are certainly correct :)